Tests/DangerTests/GitDiffTests.swift

```diff
@@ -52,28 +71,34 @@ final class GitDiffTests: XCTestCase {
                                    FileDiff.Line(text: "", changeType: .added),
                                    FileDiff.Line(text: "", changeType: .added),
                                    FileDiff.Line(text: "protocol SPMCoverageParsing {", changeType: .unchanged),
-                                   FileDiff.Line(text: "    static func coverage(spmCoverageFolder: String, files: [String]) throws -> Report", changeType: .removed),
+                                   FileDiff.Line(text: "    static func coverage(spmCoverageFolder: String, files: [String]) throws -> Report",
+                                                 changeType: .removed),
                                    FileDiff.Line(text: "}", changeType: .removed),
-                                   FileDiff.Line(text: "    static func coverage(spmCoverageFolder: String, files: [String]) throws -> Re", changeType: .added),
+                                   FileDiff.Line(text: "    static func coverage(spmCoverageFolder: String, files: [String]) throws -> Re",
+                                                 changeType: .added),
                                    FileDiff.Line(text: "", changeType: .unchanged),
                                    FileDiff.Line(text: "enum SPMCoverageParser: SPMCoverageParsing {", changeType: .unchanged),
                                    FileDiff.Line(text: "    enum Errors: Error {", changeType: .unchanged),
                                ]),
-                               .init(oldLineStart: 22, oldLineSpan: 6, newLineStart: 25, newLineSpan: 10, lines: [
+                                        .init(oldLineStart: 22, oldLineSpan: 6, newLineStart: 25, newLineSpan: 10, lines: [
                                    FileDiff.Line(text: "        let coverage = try JSONDecoder().decode(SPMCoverage.self, from: data)", changeType: .unchanged),
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitDiffTests.swift;
  Line: 84;
-->

- :warning: Line should be 140 characters or less: currently 160 characters (`line_length`)